Lib/test/test_weakref.py

```diff
@@ -285,6 +285,18 @@ def __ifloordiv__(self, other):
         p //= 5
         self.assertEqual(p, 21)
 
+    def test_proxy_matmul(self):
+        class C:
+            def __matmul__(self, other):
```

Comment:
Maybe add a test for `__rmatmul__` as well?

Reply:
Good plan. Done!